app/src/main/java/com/sapuseven/untis/data/databases/UserDatabase.kt

```diff
@@ -36,7 +37,7 @@ class UserDatabase private constructor(context: Context) : SQLiteOpenHelper(cont
 	}
 
 	override fun onCreate(db: SQLiteDatabase) {
-		db.execSQL(UserDatabaseContract.Users.SQL_CREATE_ENTRIES_V5)
+		db.execSQL(UserDatabaseContract.Users.SQL_CREATE_ENTRIES_V6)
```

Comment:
The existing `SQL_CREATE_ENTRIES_V5` was actually unintentional. It should match the current `DATABASE_VERSION`.
```suggestion
		db.execSQL(UserDatabaseContract.Users.SQL_CREATE_ENTRIES_V7)
```